### PR TITLE
Folder Added: JavaScript Assignment-3

### PR DESCRIPTION
**Pull request by akash19972306**

### CODE REVIEW

File: akash1/JavaScript/ASSIn3/Ass1.html [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+        </script>
+    </head>
+    <body>
+        <div class="container text-white border-primary bg-dark">
+            <h1 style="text-align: center"class="mt-3">Leap Year</h1>
+            <hr style="background-color: white" />
+                <div class="row">
+                    <div class="col">
+                        Enter Year:<input type="text" class="form-control" placeholder="Enter Year" id="input1"><br>
+
```
Comment (Pawan1005) [inline akash1/JavaScript/ASSIn3/Ass1.html]: remove extra line

File: akash1/JavaScript/ASSIn3/Ass1.html [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+                crossorigin="anonymous"
+            ></script>
+            <script
+                src="https://cdn.jsdelivr.net/npm/bootstrap@4.6.0/dist/js/bootstrap.bundle.min.js"
+                integrity="sha384-Piv4xVNRyMGpqkS2by6br4gNJ7DXjqk09RmUpJ8jgGtD7zP9yug3goQfGII0yAns"
+                crossorigin="anonymous"
+            ></script>
+        </div>
+    </body>
+</html>
```
Comment (Pawan1005) [inline akash1/JavaScript/ASSIn3/Ass1.html:69]: remove extra lines

**Review by Pawan1005 — CHANGES_REQUESTED** [review]

File: akash1/JavaScript/ASSIn3/Assq2.html [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+             top: 149px;
+             position: relative;
+            border-top-left-radius: 10px;
+            border-top-right-radius:10px ;
+            width: 50%;
+        }    
+        </style>
+        <script language="Javascript">
+            function isPrime(){
+           prime=parseInt(document.getElementById("input1").value);
```
Comment (Pranjali1999) [inline akash1/JavaScript/ASSIn3/Assq2.html]: Wrong Indentation.

File: akash1/JavaScript/ASSIn3/Assq2.html [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+           if(prime==1){
+            document.getElementById("p").innerHTML="It is not a prime nor composite number " +prime;
```
Comment (Pranjali1999) [inline akash1/JavaScript/ASSIn3/Assq2.html]: Wrong Indentation.

File: akash1/JavaScript/ASSIn3/Assq2.html [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+           }else if(prime<1){
+            document.getElementById("p").innerHTML="It is not a prime number " +prime;
```
Comment (Pranjali1999) [inline akash1/JavaScript/ASSIn3/Assq2.html]: Wrong Indentation.

File: akash1/JavaScript/ASSIn3/Assq2.html [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+           }else{
+               for(var i=2; i<prime; i++){
+                   if(prime%i==0){
+                    document.getElementById("p").innerHTML="It is not a prime number " +prime;
```
Comment (Pranjali1999) [inline akash1/JavaScript/ASSIn3/Assq2.html:36]: Wrong Indentation.

File: akash1/JavaScript/ASSIn3/Assq2.html [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+                       break;
+                   }
+                   else{
+                    document.getElementById("p").innerHTML="It is a prime number " +prime;
```
Comment (Pranjali1999) [inline akash1/JavaScript/ASSIn3/Assq2.html]: Wrong Indentation.

File: akash1/JavaScript/ASSIn3/Assq3.html [hunk 8]
```diff
@@ -0,0 +1,69 @@
...
+            border-top-left-radius: 10px;
+            border-top-right-radius:10px ;
+            width: 50%;
+        }    
+        </style>
+        <script language="Javascript">
+            function Even(){
+           a=parseInt(document.getElementById("input1").value);
+           if(a%2==0){
+            document.getElementById("p").innerHTML="It is Even Number ";
```
Comment (Pranjali1999) [inline akash1/JavaScript/ASSIn3/Assq3.html]: Wrong Indentation.

File: akash1/JavaScript/ASSIn3/Assq4.html [hunk 9]
```diff
@@ -0,0 +1,76 @@
...
+             top: 149px;
+             position: relative;
+            border-top-left-radius: 10px;
+            border-top-right-radius:10px ;
+            width: 50%;
+        }    
+        </style>
+        <script language="Javascript">
+            function Factorial(){
+           a=parseInt(document.getElementById("input1").value);
```
Comment (Pranjali1999) [inline akash1/JavaScript/ASSIn3/Assq4.html]: Wrong Indentation.

File: akash1/JavaScript/ASSIn3/Assq4.html [hunk 10]
```diff
@@ -0,0 +1,76 @@
...
+           var fact =1;
+           if(a == 0){
+               alert("The factorial of number is 1")
+           }
+           else if(a<0){
+            alert("The factorial of -ve number is not possible")
```
Comment (Pranjali1999) [inline akash1/JavaScript/ASSIn3/Assq4.html]: Wrong Indentation.

File: akash1/JavaScript/ASSIn3/Assq5.html [hunk 11]
```diff
@@ -0,0 +1,70 @@
...
+        .container{
+            border-top-left-radius: 10px;
+            border-top-right-radius:10px ;
+            width: 50%;
+            box-shadow: 0 10px 12px 0 rgba(213, 215, 216, 0.973), 0 6px 20px 0 rgba(206, 14, 14, 0.918);
+        }    
+        </style>
+        <script language="Javascript">
+            function Square(){
+           var row=parseInt(document.getElementById("input1").value);
```
Comment (Pranjali1999) [inline akash1/JavaScript/ASSIn3/Assq5.html]: Wrong Indentation.

**Review by Pranjali1999 — CHANGES_REQUESTED** [review]

**Review by Pranjali1999 — APPROVED** [review]

**Review by Pawan1005 — APPROVED** [review]

**Pull request merged**